Apps/Admin/Client/Components/Dashboard/RatingSummary.razor

```diff
@@ -22,14 +22,14 @@
                     <MudProgressLinear Value="@percentage" />
                 </MudItem>
                 <MudItem xs="2" lg="1" Class="d-flex justify-center">
-                    <MudText Typo="Typo.subtitle1">@count</MudText>
+                    <MudText Typo="Typo.subtitle1">@FormatNumber(@count)</MudText>
```

Comment:
Only the first @ symbol is needed to jump into C# code - this can be simplified to `@FormatNumber(count)`.